include/mrdocs/Metadata/Using.hpp

```diff
@@ -33,6 +33,9 @@ struct UsingInfo
     /** The symbols being "used". */
     std::vector<SymbolID> UsingSymbols;
 
+    /** Name information for the using declaration/directive. */
+    std::unique_ptr<NameInfo> UsingName;
```

Comment:
We already store the terminal name (i.e. `x` in `using A::x`) in `Info::Name`. This should be renamed to `Qualifier` and only store the _nested-name-specifier_ in the using declaration/using directive (i.e. the `A::` part).